js/bit4you.js

```diff
@@ -0,0 +1,624 @@
+            parsedData.timestamp = response.open_time;
+            parsedData.lastTradeTimestamp = response.update_time;
+            if (response.isOpen) {
+                parsedData.status = 'open';
+            } else {
+                parsedData.status = 'closed';
+            }
+            parsedData.symbol = response.market.replace ('-', '/');
+            if (response.requested_rate) {
+                parsedData.type = 'limit';
+            } else {
+                parsedData.type = 'market';
+            }
+            parsedData.timeInForce = undefined;
+            parsedData.side = response.type;
+            parsedData.price = response.quantity;
+            parsedData.average = response.quantity;
+            parsedData.amount = response.base_quantity;
+            parsedData.filled = response.base_quantity;
+            parsedData.remaining = response.remaining.quantity || undefined;
+            parsedData.cost = parseFloat (parsedData.filled * parsedData.price);
+            parsedData.trades = response.position;
+            parsedData.fee = {
+                'currency': response.fee.iso,
+                'cost': response.fee.quantity,
+                'rate': undefined,
+            };
+            parsedData.info = response;
+            return parsedData || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        const request = params;
+        try {
+            const response = await this.privatePostOrderPending (this.extend (request));
+            return response || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        const request = params;
+        try {
+            const response = await this.privatePostPortfolioHistory (request);
+            return response || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        // side type be limit or Market
+        const request = {};
+        request.txid = id;
+        try {
+            const response = await this.privatePostOrderCancel (request);
+            return response || [];
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        // side type be limit or Market
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const iso = market['symbol'].replace ('/', '-');
+        const request = {};
+        // if no rate is provided, the order will be executed at the current market rate
+        if (type === 'limit' && price) {
+            request.rate = price;
+        }
+        request.market = iso;
```

Comment:
You can't access dicts this way, although it works for Javascript will break the transpiled versions in Python and Php, please check out the contributing section: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys